packages/tests.backcompat/src/client-backcompat.ts

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import chalk from "chalk";
+import consola from "consola";
+import { execa } from "execa";
+import { findUpSync } from "find-up";
+import { promises as fs } from "fs";
+import * as path from "node:path";
+import { fileURLToPath } from "node:url";
+import semver from "semver";
+
+const THIS_FILE_DIR = path.dirname(fileURLToPath(import.meta.url));
+const pnpmLockFile = findUpSync("pnpm-lock.yaml");
+
+if (!pnpmLockFile) {
+  throw new Error("pnpm-lock.yaml not found. :(");
+}
+
+const rootPath = path.dirname(pnpmLockFile);
+
+const monorepoTranspilePath = path.join(
+  rootPath,
+  "packages",
+  "monorepo.tool.transpile",
+  "bin",
+  "transpile2.mjs",
+);
+
+const builtSdksPath = path.join(
+  THIS_FILE_DIR,
+  "codegen",
+  "sdks",
+);
```

Comment:
So when this runs we are codegening into `build/esm/codegen/sdks`? I think you mean to go up a directory to hit the `build/codegen` thing I mentioned?